src/i18n/locales/en/translation.json

```
@@ -177,6 +177,7 @@
     "feedback_invalid_amount": "Please provide a valid amount.",
     "toggle_coinjoin": "Send as collaborative transaction for improved privacy",
     "button_send": "Send",
+    "button_send_without_improved_privacy": "Send – without improved privacy",
```

```suggestion
    "button_send_without_improved_privacy": "Send without privacy",
```

I'd just do this. Other suggestions @dergigi?

---

I think we need the "improved" because "without privacy" isn't necessarily accurate 😅 

What about:

> "Send without privacy improvement"